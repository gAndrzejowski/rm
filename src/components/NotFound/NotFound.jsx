// @flow
import React from 'react';
import injectStylesheet from 'react-jss';
import styles from './NotFound.styles';

const NotFound = ({ classes } :Object) => (
  <article className={classes.notfound}>
    <h2>We couldn{'\''}t find this page...</h2>
    <img src="img/404.jpg" alt="" />
    <p>Try another url or use the search box.</p>
  </article>
);
NotFound.defaultProps = {
  classes: {},
};
export default injectStylesheet(styles)(NotFound);
