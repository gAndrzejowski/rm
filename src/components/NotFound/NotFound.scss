.notfound{
  flex-flow: column nowrap;
  display:flex;
  justify-content: center;
  align-items: center;
  background:white;
  padding:2em 5vw;
  min-height:50vh;
}
