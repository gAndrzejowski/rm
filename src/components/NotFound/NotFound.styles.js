const styles = {
  notfound: {
    flexFlow: 'column nowrap',
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
    background: 'white',
    padding: ['2em', '5vw'],
    minHeight: '50vh',
  },
};
export default styles;
