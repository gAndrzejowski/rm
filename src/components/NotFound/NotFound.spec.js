import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NotFound from './NotFound';

describe('404 page component', () => {
  it('renders correctly', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<NotFound />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
