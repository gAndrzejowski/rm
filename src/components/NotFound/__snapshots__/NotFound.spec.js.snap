// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page component renders correctly 1`] = `
<NotFound
  classes={
    Object {
      "notfound": "NotFound-notfound-0-1-1",
    }
  }
/>
`;
